lib/wopinator/discovery.rb

```diff
@@ -104,6 +107,24 @@ def parse_url(url)
       Addressable::URI.parse(url.gsub(/&#38;/, '&').gsub(/<.*?=.*?>/, '').gsub(/(\?|&)$/, ''))
     end
 
+    def filter_action_query_params(url, params)
+      return {} unless params.any?
+
+      supported = extract_supported_action_query_params(url)
+
+      {}.tap do |hash|
+        params.each do |k, v|
+          if key = supported[k]
+            hash[key] = v
+          end
+        end
+      end
+    end
+
+    def extract_supported_action_query_params(url)
```

Comment:
@brandonc Added this to the code as well: https://github.com/kapost/wopinator/commit/84e2fe52cbc09e203e8f3f7711a6fd5a0db389fb